src/main/resources/assets/mythicmetals/lang/de_de.json

```diff
@@ -0,0 +1,1146 @@
+{
+  "block.alloy_forgery.adamantite_block_forge_controller": "Adamantit Block Schmiede Kontrolleinheit",
+  "block.mythicmetals.adamantite_anvil": "Adamantit Amboss",
+  "block.mythicmetals.adamantite_block": "Adamantit Block",
+  "block.mythicmetals.adamantite_ore": "Adamantit Erz",
+  "block.mythicmetals.aquarium_anvil": "Aquarium Amboss",
+  "block.mythicmetals.aquarium_block": "Aquarium Block",
+  "block.mythicmetals.aquarium_glass": "Aquarium Glass",
+  "block.mythicmetals.aquarium_resonator": "Aquarium Resonator",
+  "block.mythicmetals.aquarium_steward": "Aquarium Steward",
+  "block.mythicmetals.aquarium_ore": "Aquarium Erz",
+  "block.mythicmetals.banglum_anvil": "Banglum Amboss",
+  "block.mythicmetals.banglum_block": "Banglum Block",
+  "block.mythicmetals.banglum_nuke_core": "Banglum Nuke Kern",
```

Comment:
This should instead be something akin to "Banglumatomkern", or "Atomkern (Banglum)". 
Alternatively go with something more similar to "Nuklearkern" for the Nuke Core part of the translation. Nuke Kern is not sufficient.